src/main/java/core/basesyntax/converter/DataConverterImpl.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.converter;
+
+import core.basesyntax.transaction.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> converterToTransaction(List<String> inputData) {
+        List<FruitTransaction> fruitTransactions = new ArrayList<>();

```

Comment:
It seems like your class `DataConverterImpl` is doing more than one thing. It's both parsing the input data and creating `FruitTransaction` objects. According to the Single Responsibility Principle (SRP) of SOLID, a class should have only one reason to change. Consider separating these responsibilities into different classes or methods. For example, you could have a `DataParser` class that handles the parsing of the input data, and a `TransactionFactory` class that creates `FruitTransaction` objects.